apps/services/user-profile/src/environments/environment.ts

```diff
@@ -1,3 +1,18 @@
-export const environment = {
+export default {
   production: false,
+  email: {
+    fromEmail: 'olafur@sendiradid.is',
+    fromName: 'Olafur',
+    replyToEmail: 'olafur@sendiradid.is',
+    replyToName: 'Ólafur Björn Magnússon',
+    currentBaseUrl: 'http://localhost:4200',
```

Comment:
is this fine by you being here? doesn't really matter :D

Reply:
well fine for development purposes. dooesnt actually send an email. Just a preview link